backend/utils/resource-utils.js

```diff
@@ -209,13 +209,17 @@ const updatePath = R.curry((pth, fn, obj) =>
   R.hasPath(pth, obj) ? R.assocPath(pth, fn(R.path(pth, obj)), obj) : obj
 );
 
+const isAbsoluteURL = (url) => /^((http|https|ftp):\/\/)/.test(url);
+const prefixWithHTTP = (url) => (isAbsoluteURL(url) ? url : `http://${url}`);
```

Comment:
https if no prefix maybe?

Reply:
Good point — I was originally doing this for legacy support based on [this](https://security.stackexchange.com/q/81801/163101) discussion, especially for personal sites that may be (relatively more often) still using HTTP, but after rethinking it, I agree that we should default to HTTPS since it looks like a much higher percentage of sites support it now in 2020.